versions/3.0.md

```diff
@@ -2020,7 +2020,7 @@ href: 'http://colors.my-server.com/colors/green'
 href: 'http://colors.my-server.com/colors/blue'
 ```
 
-As with all links, it at the the clients' discretion to follow them, and permissions and the ability to make a successful call to that link is not guaranteed solely by the existence of a relationship.
+As with all links, it is at the the clients' discretion to follow them, and permissions and the ability to make a successful call to that link is not guaranteed solely by the existence of a relationship.
```

Comment:
@willdady good eyes. In fact I think the whole sentence needs rewording. Was looking too microscopically.